apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ExecuteHelper.cs

```diff
@@ -14,14 +14,18 @@
 
 using Grpc.Core;
 using System;
-using System.Linq;
 using System.Threading.Tasks;
-using Google.Api.Gax;
 
 namespace Google.Cloud.Spanner.V1
 {
-    internal static class ExecuteHelper
+    /// <summary>
+    /// Helper class to wrap operations.
+    /// </summary>
+    public static class ExecuteHelper
     {
+        /// <summary>
+        /// Waits for <paramref name="task"/> to complete, handling session expiry by marking the session appropriately.
+        /// </summary>
         public static async Task<T> WithSessionChecking<T>(this Task<T> task, Func<Session> sessionFunc)
```

Comment:
Not that I can remember, at the moment. But we could potentially add something in `CallSettings`. @chrisdunelm any thoughts on that?